sql/core/src/test/resources/sql-tests/inputs/collations.sql

```diff
@@ -90,3 +90,14 @@ select 'a' collate en_ci_ai = 'Å';
 select 'Kypper' collate sv < 'Köpfe';
 select 'Kypper' collate de > 'Köpfe';
 select 'I' collate tr_ci = 'ı';
+
+-- create table for str_to_map
+create table t1 (text string collate utf8_binary, pairDelim string collate utf8_lcase, keyValueDelim string collate utf8_binary) using parquet;
```

Comment:
```suggestion
create table t2 (text string collate utf8_binary, pairDelim string collate utf8_lcase, keyValueDelim string collate utf8_binary) using parquet;
```
let's just use another table name here, so that we don't confuse it with `t1` (above) in the future